packages/trpc/server/routers/viewer/aiVoiceAgent/create.handler.ts

```diff
@@ -12,16 +13,21 @@ type CreateHandlerOptions = {
 };
 
 export const createHandler = async ({ ctx, input }: CreateHandlerOptions) => {
-  const { teamId, name, workflowStepId, ...retellConfig } = input;
+  const { teamId, name, workflowStepId, templateWorkflowId, ...retellConfig } = input;
 
   const aiService = createDefaultAIPhoneServiceProvider();
 
+  const generalPrompt = templateWorkflowId
+    ? calAIPhoneWorkflowTemplates?.[templateWorkflowId as keyof typeof calAIPhoneWorkflowTemplates]
```

Comment:
```suggestion
    ? calAIPhoneWorkflowTemplates.[templateWorkflowId as keyof typeof calAIPhoneWorkflowTemplates]
```

Don't think _calAIPhoneWorkflowTemplates_ is nullish